lib/chef/knife/client_bulk_delete.rb

```diff
@@ -71,7 +71,7 @@ def check_and_delete_validators(validators)
           else
             ui.msg("The following validators will be deleted:")
             print_clients(validators)
-            if ui.confirm_without_exit("Are you sure you want to delete these validators")
+            if ui.confirm_without_exit("Are you sure you want to delete these validators?")
```

Comment:
I believe all of these confirm methods already append the "?": https://github.com/chef/chef/blob/master/lib/chef/knife/core/ui.rb#L220